### PR TITLE
Cloud SQL doc fixes.

### PR DESCRIPTION
**Pull request by dlorenc**
A few miscellaneous fixes to the cloud sql sample.

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @jeffmendoza

File: cloudsql/README.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ for how to get started. As noted above, if you have a Google Container Engine cl
...
-See the [Cloud SQL getting started](https://cloud.google.com/sql/docs/access-control) page for more information.
+See the [Cloud SQL access control](https://cloud.google.com/sql/docs/access-control) page for more information.
 
 #### Create Secrets
 
 You'll need to create several `Secret` resources to allow the SQL proxy to connect with your SQL instance. First, you'll need to
 create a secret resource containing the Service Account credentials to allow the proxy to communicate with the Cloud SQL API.
 
-First, download the JSON Service Account credentials.
+First, create download the JSON Service Account credentials following [these steps](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#creatinganaccount).
```
Comment (jeffmendoza) [inline cloudsql/README.md]: I think you meant to remove "download"

**Comment by jeffmendoza** [comment]
Sorry for the delay! Looks like I forgot to watch my own repo =/

File: cloudsql/README.md (hunk 1 above)
Comment (dlorenc) [inline cloudsql/README.md]: Nice catch, I think I meant to have both :)

**Comment by jeffmendoza** [comment]
LGTM

**Pull request merged**